core/src/main/scala/org/apache/spark/internal/config/package.scala

```diff
@@ -1097,9 +1097,9 @@ package object config {
 
   private[spark] val SHUFFLE_HOST_LOCAL_DISK_READING_ENABLED =
     ConfigBuilder("spark.shuffle.readHostLocalDisk.enabled")
-      .doc("If enabled, shuffle blocks requested from those block managers which are running on " +
-        "the same host are read from the disk directly instead of being fetched as remote blocks " +
-        "over the network.")
+      .doc("If enabled (and `spark.shuffle.useOldFetchProtocol` is disabled), shuffle blocks " +
```

Comment:
`${SHUFFLE_USE_OLD_FETCH_PROTOCOL.key}`

Reply:
Transforming the first part to string interpolation has not been forgotten.